centinel/models.py

```diff
@@ -77,13 +79,6 @@ def __init__(self, **kwargs):
         country = kwargs.get('country')
         if country is not None and (len(country) == 2):
             self.country = country
-        # if we have a date for the last time the client was seen,
-        # then add it, otherwise, we won't add the last seen date in
-        # case we need to remove it for security reasons or traveling
-        if 'last_seen' in kwargs:
-            self.last_seen = kwargs['last_seen']
```

Comment:
No we're not. This is to make sure that the code won't crash in some cases where it has not been set (calling `date` for None-type object).
The value is set in `update_client_info()`.